src/main/java/jp/co/future/uroborosql/SqlAgentImpl.java

```diff
@@ -1859,9 +1885,6 @@ private void transformContext(final ExecutionContext executionContext) {
 		var sqlName = executionContext.getSqlName();
 		if (ObjectUtils.isEmpty(originalSql) && getSqlResourceManager() != null) {
 			originalSql = getSqlResourceManager().getSql(sqlName);
-			if (ObjectUtils.isEmpty(originalSql)) {
```

Comment:
This logic has been removed because an exception was thrown in SqlResourceManager#getSql